tasmota/tasmota_xsns_sensor/xsns_18_pms5003.ino

```diff
@@ -27,6 +27,8 @@
  * You can either support PMS3003 or PMS5003-7003 at one time. To enable the PMS3003 support
  * you must enable the define PMS_MODEL_PMS3003 on your configuration file.
  * For PMSx003T models that report temperature and humidity define PMS_MODEL_PMS5003T
+ * This module can also support de Winsum ZH03x series of dust particle sensors,
```

Comment:
It's ``Winsen``. Not Winsum, places in the Netherlands (yes there are more than one)

Reply:
Done, Thank you for the comments.